hvac/api/secrets_engines/transit.py

```diff
@@ -603,7 +603,7 @@ def generate_data_key(
         )
 
     def generate_random_bytes(
-        self, n_bytes=None, output_format=None, mount_point=DEFAULT_MOUNT_POINT
+        self, n_bytes=None, output_format=None, source="", mount_point=DEFAULT_MOUNT_POINT
```

Comment:
`source` must be moved to the end of the parameter list so that it doesn't break clients using positional arguments. We should also default it to `None` and then the method needs to handle that by not templating the value into the URL.